posts/2022-06-16-http-registry-testing.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Cargo http-registry call for testing"
+author: Arlo Siemsen
+team: The Cargo Team <https://www.rust-lang.org/governance/teams/dev-tools#cargo>
+---
+
+The Cargo [`http-registry`][http-registry] feature is ready for testing. The feature causes Cargo
+to access the crates.io index over http, rather than git. It can provide a significant
+performance improvement, especially if the git index is out-of-date or not yet cloned.
+
+To try it out, add the `-Zhttp-registry` flag on the `nightly-2022-06-09` or newer
+build of Cargo. For example, to update dependencies:
+
+```
+rustup update nightly
+cargo +nightly -Zhttp-registry update
+```
+
+You can leave feedback on the [internals thread][internals].
+
+Accessing the index over http allows crates.io to continue growing without hampering
+performance. The improvement should be especially noticeable in CI environments, where
```

Comment:
Allows the number of crates to grow? The number of pulls? Similarly, the performance for crates.io or for clients?